src/psyclone/tests/gocean1p0_stencil_test.py

```diff
@@ -0,0 +1,227 @@
+# -----------------------------------------------------------------------------
+# BSD 3-Clause License
+#
+# Copyright (c) 2018, Science and Technology Facilities Council
```

Comment:
I don't know the answer to this but should we keep 2017-2018 here, in line with the other source files (i.e. we own the Copyright for PSyclone from 2017- and this file is part of PSyclone)?

Reply:
I don't know either but I assume we can't copyright code that does not exist yet so I think 2018 is correct!